packages/dispatchr/package.json

```diff
@@ -8,9 +8,9 @@
     "url": "git://github.com/yahoo/fluxible.git"
   },
   "scripts": {
-    "test": "../../node_modules/.bin/mocha tests/unit/",
-    "cover": "BABEL_ENV=test ../../node_modules/.bin/nyc ../../node_modules/.bin/_mocha tests/unit/",
-    "lint": "../../node_modules/.bin/eslint lib/ addons/ utils/ index.js"
+    "test": "mocha tests/unit/",
```

Comment:
we've always used the .. notation because if you try to run the command in the package folder it fails since npm can't find the binary locally.

Reply:
> Changes look good in general, not sure if we need to change the run scripts in each package.json. Are you just executing the commands from the repository root?

You're completely right about not being able to run the commands inside each package with this patch.

I didn't realize it because I was always running `lerna test` (which I do most of the time since the tests are fast and one package can break another) or `lerna test --scope package-name` (to run all the tests for a given package) or even `npx mocha packages/package-name/tests/test-file.js` (for a specific test file) always from the monorepo root.

So we have 2 alternatives:

1. Keep as it was before: polluted, but allows running tests from inside each package
2. Keep as it is: cleaner, but doesn't allow running tests inside each package (lerna, mocha and eslint allow us to specify target packages/files anyway)

I'm ok with both (we have a "clean" X "correct" trade off), but which one would you prefer?

Reply:
@pablopalacios lets go with how it was before. The syntax we had doesn't break how lerna is used at the root. It's a nice convenience for us to be able to run tests from within the package directory.

Reply:
Reverted.